loggly/winston-loggly-bulk
# LOG-11499 update dependencies for security reasons

*Task*
https://swicloud.atlassian.net/browse/LOG-11499

*Overview*
Solution for vulnerabilities:
[https://github.com/loggly/winston-loggly-bulk/issues/71](https://github.com/loggly/winston-loggly-bulk/issues/71)
[https://github.com/loggly/winston-loggly-bulk/issues/70](https://github.com/loggly/winston-loggly-bulk/issues/70)

*How to test*
- Checkout the branch locally.
- Install dependencies with `npm ci` (so the `package-lock.json` is followed).
- Check that `node_modules` directory doesn't contain the vulnerable versions anymore.
- Try to run tests via `npm run test`.
- Try to log via the library.
  - Use the library in another node project (or create a new one).
    - FYI it is possible to install `npm`dependency from disk via `npm install --save <relative-location-to-the-winston-on-disk>`
  - How to use the library? It is described in `readme.md` of the *winston-loggly-bulk*
